src/shared/sam/debugger/awsSamDebugConfiguration.ts

```diff
@@ -39,8 +40,31 @@ export function isCodeTargetProperties(props: TargetProperties): props is CodeTa
     return props.target === CODE_TARGET_TYPE
 }
 
+/**
+ * Creates a description for a SAM debugconfig entry (in launch.json).
+ *
+ * Example: `makeName('foo', '/bar/baz', 'zub')` => `"baz:foo (zub)"
```

Comment:
Just making sure, this shouldn't be `bar/baz:foo (zub)`?

Reply:
we don't want the full path, just the parent directory as a "hint"